var/spack/repos/builtin/packages/highfive/package.py

```diff
@@ -11,12 +11,12 @@ class Highfive(CMakePackage):
     """HighFive - Header only C++ HDF5 interface"""
 
     homepage = "https://github.com/BlueBrain/HighFive"
-    url      = "https://github.com/BlueBrain/HighFive/archive/v1.2.tar.gz"
+    url      = "https://github.com/BlueBrain/HighFive/archive/v2.0.tar.gz"
     giturl   = "https://github.com/BlueBrain/HighFive.git"
 
     version('develop', git=giturl)
     # todo : waiting for author to release new version
-    version('1.6', git=giturl, commit='5c4e54707879f04ce5')
+    version('2.0')
```

Comment:
add checksum : you can use `spack checksum`

```
→ spack checksum highfive
==> Found 7 versions of highfive:

  2.0  https://github.com/BlueBrain/HighFive/archive/v2.0.tar.gz
  1.5  https://github.com/BlueBrain/HighFive/archive/v1.5.tar.gz
  1.4  https://github.com/BlueBrain/HighFive/archive/v1.4.tar.gz
  1.3  https://github.com/BlueBrain/HighFive/archive/v1.3.tar.gz
  1.2  https://github.com/BlueBrain/HighFive/archive/v1.2.tar.gz
  1.1  https://github.com/BlueBrain/HighFive/archive/v1.1.tar.gz
  1.0  https://github.com/BlueBrain/HighFive/archive/v1.0.tar.gz

==> How many would you like to checksum? (default is 1, q to abort) 2
==> Downloading...
==> Fetching https://github.com/BlueBrain/HighFive/archive/v2.0.tar.gz
######################################################################## 100.0%
==> Fetching https://github.com/BlueBrain/HighFive/archive/v1.5.tar.gz
######################################################################## 100.0%
==> Checksummed 2 versions of highfive

    version('2.0', sha256='deee33d7f578e33dccb5d04771f4e01b89a980dd9a3ff449dd79156901ee8d25')
    version('1.5', sha256='f194bda482ab15efa7c577ecc4fb7ee519f6d4bf83470acdb3fb455c8accb407')
```

Reply:
I somehow thought it hadn't been released yet.